src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Conformance/ConformanceBuilderTests.cs

```diff
@@ -73,7 +73,7 @@ public void GivenAConformanceBuilder_WhenVersionofResourceIsDifferentFromDefault
             IOptions<CoreFeatureConfiguration> configuration = Substitute.For<IOptions<CoreFeatureConfiguration>>();
             Dictionary<string, string> overrides = new();
             VersioningConfiguration versionConfig = new();
-            versionConfig.ResourceTypeOverrides.Add("Patient", "no-version");
+            versionConfig.ResourceTypeOverrides.Add("patient", "no-version");
```

Comment:
I think this should be a Theory and you should try upper/lower values

Reply:
I agree, we should also test for irregular casing like "pAtIeNt"

Reply:
Thanks. I updated the test to a theory and added more test cases to validate.